### DISCUSSION TITLE
Issue with installation

### DISCUSSION THREAD

**Discussion by TBCNoob** [Lailloken/Exile-UI#352] [General]
Hi,

first of all I was looking through the UI you set up and all I have to say is I cant wait to use it much love and appreciation for the tool you have made.

Can you kindly set up a noob step by step installation guide for me please. I am having a hard time getting it installed. so far I have reached this issue, I appreciate any and all feedback
![Error ](https://github.com/Lailloken/Lailloken-UI/assets/165328826/8a1b92cf-3fd7-4243-8ee7-7f2ddf1c2d95)

**Comment by Lailloken** [comment]
- click the big download banner on the github front page
  - open the downloaded zip-file

- inside, there should be a folder "Lailloken-UI something something"
  - drag this folder onto your Desktop

- close the zip-file again

- rename the folder you dragged onto the desktop to simply "Lailloken UI", then move it somewhere where you'll easily find it (e.g. next to your PoE install)

- Launch "Lailloken UI.ahk" inside the folder

**Reply by TBCNoob** [reply]
thank you for the quick response I get this error now
![error 2](https://github.com/Lailloken/Lailloken-UI/assets/165328826/64c9ee14-7982-42f3-9e5c-6b9aefaf9a78)

**Reply by Lailloken** [reply]
That simply means you don't have the correct version of AHK installed on your system. There's also a banner for that on the front page. Make sure to get the latest 1.1 version and not a 2.x version.
`EDIT: I just noticed they changed the page and made it more complicated`

**Reply by TBCNoob** [reply]
no worries I just got it to run thank you so muchhhh
